### PR TITLE
Algorithms module edits

### LINKED ISSUE #317: Copy edit the Qiskit migration guides
Double check the content in our 3 migration guides:

* [x] https://github.com/Qiskit/documentation/blob/main/docs/api/migration-guides/qiskit-algorithms-module.mdx
* [x] https://github.com/Qiskit/documentation/blob/main/docs/api/migration-guides/qiskit-opflow-module.mdx
* [x] https://github.com/Qiskit/documentation/blob/main/docs/api/migration-guides/qiskit-quantum-instance.mdx

### PR DESCRIPTION
**Pull request by beckykd**
closes #317

### CODE REVIEW

**Comment by Eric-Arellano** [comment]
> closes https://github.com/Qiskit/documentation/issues/317

The other guides didn't need anything?

**Comment by beckykd** [comment]
> > closes #317
> 
> The other guides didn't need anything?

Oh, I'm sure they do.  I'll work on them today. :)

**Comment by beckykd** [comment]
> Ah, sg. Rather than saying "Closes #317", it's better to say "Part of #317". GitHub treats the word "closes" specially and would close the issue, even though there's follow up work.

Oh that was intentional.  I was going to wait until they were all done before closing the PR

File: docs/api/migration-guides/qiskit-opflow-module.mdx [hunk 1]
```diff
@@ -5,24 +5,22 @@ description: Stop using the deprecated `qiskit.opflow` module
 
 # Opflow Migration Guide
```
Comment (abbycross) [inline docs/api/migration-guides/qiskit-opflow-module.mdx:6]:
```suggestion
# Opflow migration guide
```

File: docs/api/migration-guides/qiskit-opflow-module.mdx [hunk 2]
```diff
@@ -583,23 +580,23 @@ Statevector([0.+0.j, 0.+0.j, 0.-1.j, 0.+0.j],
...
-| [`qiskit.opflow.converters.DictToCircuitSum`](../qiskit/qiskit.opflow.converters.DictToCircuitSum) | No direct replacement. This class was used to convert from [`qiskit.opflow.state_fns.DictStateFn`](../qiskit/qiskit.opflow.state_fns.DictStateFn)s or [`qiskit.opflow.state_fns.VectorStateFn`](../qiskit/qiskit.opflow.state_fns.VectorStateFn)s to equivalent [`qiskit.opflow.state_fns.CircuitStateFn`](../qiskit/qiskit.opflow.state_fns.CircuitStateFn)s. |
+| [`qiskit.opflow.converters.AbelianGrouper`](../qiskit/qiskit.opflow.converters.AbelianGrouper) | This class allowed a sum a of Pauli operators to be grouped. Similar functionality can be achieved through the [`qiskit.quantum_info.SparsePauliOp.group_commuting`](../qiskit/qiskit.quantum_info.SparsePauliOp#group_commuting) method of [`qiskit.quantum_info.SparsePauliOp`](../qiskit/qiskit.quantum_info.SparsePauliOp).  However, this is not a one-toone replacement, as you can see in the example [below](#example-commuting). |
```
Comment (abbycross) [inline docs/api/migration-guides/qiskit-opflow-module.mdx:596]:
```suggestion
| [`qiskit.opflow.converters.AbelianGrouper`](../qiskit/qiskit.opflow.converters.AbelianGrouper) | This class allowed a sum a of Pauli operators to be grouped. Similar functionality can be achieved through the [`qiskit.quantum_info.SparsePauliOp.group_commuting`](../qiskit/qiskit.quantum_info.SparsePauliOp#group_commuting) method of [`qiskit.quantum_info.SparsePauliOp`](../qiskit/qiskit.quantum_info.SparsePauliOp).  However, this is not a one-to-one replacement, as you can see in the example [below](#example-commuting). |
```

File: docs/api/migration-guides/qiskit-opflow-module.mdx [hunk 3]
```diff
@@ -945,18 +941,18 @@ q: ┤ U3(2,-π/2,π/2) ├
 
 ## Expectations
 
-Expectations are converters which enable the computation of the expectation value of an observable with respect to some state function.
-This functionality can now be found in the [`qiskit.primitives.Estimator`](../qiskit/qiskit.primitives.Estimator) primitive. Please remember that there
-are different `Estimator` implementations, as noted in the warning box [here](#attention_primitives)
+Expectations are converters that enable an observable's expectation value to be computed with respect to some state function.
+This functionn can now be found in the [`qiskit.primitives.Estimator`](../qiskit/qiskit.primitives.Estimator) primitive. Remember that there
```
Comment (abbycross) [inline docs/api/migration-guides/qiskit-opflow-module.mdx:945]:
```suggestion
This function can now be found in the [`qiskit.primitives.Estimator`](../qiskit/qiskit.primitives.Estimator) primitive. Remember that there
```

**Review by abbycross — APPROVED** [review]
teensy fixes, then good to go! Nice job 👍

**Pull request merged**